doc/source/how-to/documenting.rst

```diff
@@ -469,349 +470,210 @@ branch within this repository. This is done by uploading the generated
 
 
-Deploy using a PAT
-""""""""""""""""""
-To set up the documentation deployment using a PAT (personal access token), you
-must first create a PAT in the ``Settings`` section in your
-GitHub profile. In the left side bar, select the ``Developer Settings`` section
-and then ``Personal access tokens``. Finally, click ``Generate new token`` and
-give it ``write`` permissions on ``Repositories`` at least. You are then prompted
-with the value of the ``TOKEN``. Make sure to copy the value of the ``TOKEN`` as
-you are not be able to retrieve it later. Finally, click ``Configure SSO`` to
-allow using it with the PyAnsys repositories you have access to.
+With all previous configuration, your project is ready to use multi-version
+documentation in an automated way. This means that, every time you release a
+new version, it will be added to the drop-down button in the documentation page
+of the project.
 
 .. note::
 
-    In some cases, the authentication might need specific approval. If you
-    still get authentication errors like those that follow, click the link in
-    the log to authorize the workflow.
+    Only the development together with the latest three stable versions are
+    shown by default in the documentation drop-down.
 
-    .. code-block:: text
-        :emphasize-lines: 1,2,9
 
-        remote: The `pyansys' organization has enabled or enforced SAML SSO. To access
-        remote: this repository, visit https://github.com/orgs/pyansys/sso?            authorization_request=AGWYQUM5VKPHAQHRS2H3JNTCVBNE7A5PN5ZGOYLONF5GC5DJN5XF62LEZYB663VUVVRX      EZLEMVXHI2LBNRPWSZGOGU33VEFPMNZGKZDFNZ2GSYLML52HS4DFVNHWC5LUNBAWGY3FONZQ
-        remote: and try your request again.
-        fatal: unable to access 'https://github.com/pyansys/pynexus-dev-docs.git/': The requested URL returned error: 403
-        Running post deployment cleanup jobs… 🗑️
-        /usr/bin/git checkout -B github-pages-deploy-action/x00pqaqlu
-        Reset branch 'github-pages-deploy-action/x00pqaqlu'
-        /usr/bin/chmod -R 777 github-pages-deploy-action-temp-deployment-folder
-        /usr/bin/git worktree remove github-pages-deploy-action-temp-deployment-folder --force
-        Error: The deploy step encountered an error: The process '/usr/bin/git' failed with exit code 128 ❌
+Mature projects configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The term *mature* projects denotes those libraries which already released
+several versions. Developers of mature projects should apply previous changes
+in the ``main`` branch of their project and then *cherry-pick* these changes in
+the latests three stable release branches. This ensures that the multi-version
+documentation is able to function as desired.
 
+For future releases of your project, since the new ``release/X.Y`` branches are
+created from ``main`` with the new configuration, no additional changes are
+required.
```

Comment:
```suggestion
Because new ``release/X.Y`` branches are created from ``main`` branch with
the new configuration, no additional changes are required for future releases
of your project.
```